java/text/README.md

```diff
@@ -0,0 +1,41 @@
+# Text Detection using the Vision API
+
+## Introduction
+
+This example uses the [Cloud Vision API](https://cloud.google.com/vision/) to
+detect text within images, stores this text in an index, and then lets you
+query this index.
+
+## Initial Setup
+
+### Install and Start Up a Redis server
+
+This example uses a [redis](http://redis.io/) server, which must be up and
+running before you start the indexing.  To install Redis, follow the
+instructions on the [download page](http://redis.io/download), or install via a
+package manager like [homebrew](http://brew.sh/) or `apt-get` as appropriate
+for your OS.
+
+The example assumes that the server is running on `localhost`, on the default
+port, and it uses [redis
+dbs](http://www.rediscookbook.org/multiple_databases.html) 0 and 1 for its data.
+Edit the example code before you start if your redis settings are different.
+
+### Set up OpenNLP
+
+Download Tokenizer data and save it to this directory.
+
+    wget http://opennlp.sourceforge.net/models-1.5/en-token.bin
+
+## Run the sample
+
+To build and run the sample, run the following from this directory:
+
+    mvn clean compile assembly:single
+    java -cp target/text-1.0-SNAPSHOT-jar-with-dependencies.jar com.google.cloud.vision.samples.text.TextApp ../../data/text/wakeupcat.jpg
```

Comment:
Done. I actually changed this to just the directory instructions. I wrote the README before I figured out how to walk a directory in Java. The fact that it works with just a single file was unintended.